criu/kerndat.c

```diff
@@ -1065,7 +1071,8 @@ int kerndat_init(void)
 	memset(&kdat, 0, sizeof(kdat));
 
 	preload_socket_modules();
-	preload_netfilter_modules();
+	if (!opts.uid)
+		preload_netfilter_modules();
```

Comment:
CAP_SYS_MODULE or CAP_NET_ADMIN

Reply:
This actually fails with
```
openat(AT_FDCWD, "/run/xtables.lock", O_RDONLY|O_CREAT, 0600) = -1 EACCES (Permission denied)
```
so checking for uid 0 seems to be the best approach.

Reply:
Will modules be loaded without CAP_SYS_MODULE or CAP_NET_ADMIN?

Reply:
No. On my system `preload_netfilter_modules()` runs iptables and iptables will fail while trying to access `/run/xtables.lock`. I do not think any capability will make this work.